src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageAllMethod.cs

```diff
@@ -59,13 +59,25 @@ protected object CreateVirtualMachineScaleSetReimageAllDynamicParameters()
             pVMScaleSetName.Attributes.Add(new AllowNullAttribute());
             dynamicParameters.Add("VMScaleSetName", pVMScaleSetName);
 
+            var pInstanceIds = new RuntimeDefinedParameter();
+            pInstanceIds.Name = "InstanceId";
+            pInstanceIds.ParameterType = typeof(string[]);
+            pInstanceIds.Attributes.Add(new ParameterAttribute
+            {
+                ParameterSetName = "InvokeByDynamicParameters",
+                Position = 3,
+                Mandatory = false
+            });
+            pInstanceIds.Attributes.Add(new AllowNullAttribute());
+            dynamicParameters.Add("InstanceId", pInstanceIds);
+
             var pArgumentList = new RuntimeDefinedParameter();
             pArgumentList.Name = "ArgumentList";
             pArgumentList.ParameterType = typeof(object[]);
             pArgumentList.Attributes.Add(new ParameterAttribute
             {
                 ParameterSetName = "InvokeByStaticParameters",
-                Position = 3,
+                Position = 4,
```

Comment:
This is because InstanceID is added to Swagger spec for this API, and our Powershell generator added the new parameter before 'ArgumentList'.  However this part of code is for Invoke command with dynamic parameters, which we don't generate the cmdlet anymore.   We use only static parameters for generated cmdlets, so we can ignore this part.  (I think we can now remove this part of the code from the generation, but I haven't done the removal yet.)